tests/layer_tests/onnx_tests/test_pooling.py

```diff
@@ -335,77 +339,77 @@ def create_global_net(self, shape, op, ir_version):
-            pytest.skip("Pooling layer support only 4D and 5D input tensors")
-        self._test(*self.create_net(**params, op='AveragePool', ir_version=ir_version, opset=10),
-                   ie_device, precision, ir_version, temp_dir=temp_dir)
-
-    @pytest.mark.parametrize("params", test_data)
-    @pytest.mark.parametrize("st_order", [None, 1])
-    @pytest.mark.nightly
-    def test_maxpool_opset8(self, params, st_order, ie_device, precision, ir_version, temp_dir):
-        if not len(params['shape']) in [4, 5]:
-            pytest.skip("Pooling layer support only 4D and 5D input tensors")
-        self._test(*self.create_net(**params, op='MaxPool', storage_order=st_order, ir_version=ir_version, opset=8),
-                   ie_device, precision, ir_version, temp_dir=temp_dir)
-
-    @pytest.mark.parametrize("params", test_data_autopad)
-    @pytest.mark.nightly
-    def test_maxpool_opset8_autopad(self, params, ie_device, precision, ir_version, temp_dir):
-        if not len(params['shape']) in [4, 5]:
-            pytest.skip("Pooling layer support only 4D and 5D input tensors")
-        self._test(*self.create_net(**params, op='MaxPool', ir_version=ir_version, opset=8),
-                   ie_device, precision, ir_version, temp_dir=temp_dir)
-
-    @pytest.mark.parametrize("params", test_data)
-    @pytest.mark.parametrize("st_order", [None, 1])
-    @pytest.mark.parametrize("ceil", [True, False])
-    @pytest.mark.nightly
-    def test_maxpool_opset10(self, params, st_order, ceil, ie_device, precision, ir_version, temp_dir):
-        if not len(params['shape']) in [4, 5]:
-            pytest.skip("Pooling layer support only 4D and 5D input tensors")
-        self._test(*self.create_net(**params, op='MaxPool', storage_order=st_order, ceil=ceil, ir_version=ir_version,
-                                    opset=10), ie_device, precision, ir_version, temp_dir=temp_dir)
-
-    @pytest.mark.parametrize("params", test_data_autopad_precommit)
-    @pytest.mark.precommit
-    def test_maxpool_opset10_autopad(self, params, ie_device, precision, ir_version, temp_dir):
-        if not len(params['shape']) in [4, 5]:
-            pytest.skip("Pooling layer support only 4D and 5D input tensors")
-        self._test(*self.create_net(**params, op='MaxPool', ir_version=ir_version, opset=10),
-                   ie_device, precision, ir_version, temp_dir=temp_dir)
+    # @pytest.mark.parametrize("params", test_data)
+    # @pytest.mark.parametrize("incl_pad", [None, 1])
+    # @pytest.mark.nightly
+    # def test_avgpool_opset7(self, params, incl_pad, ie_device, precision, ir_version, temp_dir):
+    #     if not len(params['shape']) in [4, 5]:
+    #         pytest.skip("Pooling layer support only 4D and 5D input tensors")
+    #     self._test(
+    #         *self.create_net(**params, op='AveragePool', count_include_pad=incl_pad, ir_version=ir_version, opset=7),
+    #         ie_device, precision, ir_version, temp_dir=temp_dir)
+    #
```

Comment:
Is this commented code needed? If no, maybe it should be deleted? Or this code was commented for debugging?

Reply:
It was commented for debugging, I will fix it

Reply:
Done